src/connectors/hits/__tests__/connectHits-test.js

```diff
@@ -3,7 +3,7 @@ import { TAG_PLACEHOLDER } from '../../../lib/escape-highlight';
 import connectHits from '../connectHits';
 
 jest.mock('../../../lib/utils/hits-absolute-position', () => ({
-  addAbsolutePosition: hits => hits,
+  addAbsolutePosition: hits => hits.map(x => x),
```

Comment:
I changed this so that my test would fail before the modification, to keep it closer to the real implementation

Reply:
We should add a comment explaining why we map because out of context, it doesn't make much sense (→ to create a new array instance). It will also make it explicit that this part can be removed once we change the implementation.